---
title: "Final_Project"
author: "Christopher_Zdyrski"
date: "4/13/2021"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

###Load in librarys

```{r}
library(Rsubread)
library(readr)
```


###Download transcriptome and raw reads
### From the A. l. cowlesi
```{r}
read_csv("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE129581&format=file&file=GSE129581%5Fbc%5Ftranscriptome%5Fscaffolds%2Ecsv%2Egz") -> d

snake_transciptome <- read.table('https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129581/suppl/GSE129581_transcriptome-assembly_min500bp.fna.gz', header=TRUE, sep="\t",stringsAsFactors = FALSE)

raw_reads <- read.table('https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE129581&format=file', header=TRUE, sep="\t",stringsAsFactors = FALSE)
```


###Look at dimensions of the two files

```{r}
head(snake_transciptome)
dim(snake_transciptome)
str(snake_transciptome)

head(raw_reads)
dim(raw_reads)
str(raw_reads)
```


###Align the reads to the transcriptome

```{r}
args(align)
```

###Quality Control of reads
###Use qualityScores from Rsubread library
```{r}
qs_aligned_reads <- qualityScores(filename="aligned_reads",nreads=100)
head(qs_aligned_reads)
```

###Filter for genes of interest

```{r}


```

###Make Heatmap

```{r}


```


