---
title: "Final_Project"
author: "Christopher_Zdyrski"
date: "4/13/2021"
output: html_document
---
#This file is to make the RNASeq heatmap found in the Post et al. paper figure S.2F.
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

#Install packages
```{r}
install.packages('R.utils')
install.packages('data.table')
```

#Load in librarys
```{r}
library(readr)
library(data.table)
```

#Download transcriptome and raw reads
#From the A. l. cowlesi
#Decide what files we need
```{r}
#This is the transcriptome we will align too
raw_assembled_transciptome = fread("https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129581/suppl/GSE129581_transcriptome-assembly_min500bp.fna.gz")

#File for organoids
tissue_raw_reads = fread("https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129581/suppl/GSE129581_Tissue_allPlates.csv.gz")

#File for cell date
raw_cell_reads = fread("https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129581/suppl/GSE129581_Cellseq2barcodes.csv.gz")
```

###Look at dimensions of the files
```{r}
#Analyze the transcriptome file
head(raw_transciptome)
dim(raw_transciptome)
str(raw_transciptome)

#Analyze the tissue raw reads file
head(tissue_raw_reads)
dim(tissue_raw_reads)
str(tissue_raw_reads)

#Analyze the transcriptome file
head(raw_reads)
dim(raw_reads)
str(raw_reads)
```


###Build the index
```{r}
buildindex(basename="chr1_mm10",reference="chr1.fa")
```



###May need these later
```{r}
install.packages('Rsubread')
library(Rsubread)
library(readr)
library(data.table)
```

###Align the reads to the transcriptome

```{r}
args(align)
```

###Quality Control of reads
###Use qualityScores from Rsubread library
```{r}
qs_aligned_reads <- qualityScores(filename="aligned_reads",nreads=100)
head(qs_aligned_reads)
```

###Filter for genes of interest

```{r}


```

###Make Heatmap

```{r}


```


